index.html

```diff
@@ -250,7 +249,40 @@ <h3 class="title">Project Tile goes here</h3>
                 </div>
             </div>
         </section>
+
+        <footer>
+            <section class="information-footer">
+                <div class="logo-footer">
+                    <a href="index.html"><img src="images/logo 1 (2).svg" alt="logo"></a>
+                </div>
+
+                <div class="infor">
```

Comment:
Acho que um nome melhor pra essa sessão seria `content-footer`
```suggestion
                <div class="content-footer">
```

Reply:
feito!